pandas/indexes/api.py

```diff
@@ -0,0 +1,116 @@
+from .base import (Index, _new_Index,  # noqa
+                   _ensure_index, _get_na_value,
+                   InvalidIndexError)
+from .category import CategoricalIndex  # noqa
+from .multi import MultiIndex  # noqa
+from .numeric import NumericIndex, Float64Index, Int64Index  # noqa
+from .range import RangeIndex  # noqa
+
+import pandas.core.common as com
+import pandas.lib as lib
+
+
+__all__ = ['Index', 'MultiIndex', 'NumericIndex', 'Float64Index', 'Int64Index',
+           'CategoricalIndex', 'RangeIndex',
+           'InvalidIndexError',
+           '_new_Index',
+           '_ensure_index', '_get_na_value', '_get_combined_index',
+           '_get_distinct_indexes', '_union_indexes',
```

Comment:
any reason you are expliciting adding the 'private' index methods to the import space?